guides/v2.3/config-guide/cookie-reference.md

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
You can add new topics to the site navigation by adding an entry to the TOC file

> _data/toc/configuration-guide.yml